### DISCUSSION TITLE
Page not showing the theme

### DISCUSSION THREAD

**Discussion by big616** [community/community#22033] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all, my page at <a href="http://big616.github.io" rel="nofollow noopener">http://big616.github.io</a> isn’t showing my theme. I read all the documentation, and couldn’t find why. Already spend a day looking and now i’m calling for helpppp. I went thru the the set up, but can’t seem to pin point my problem. I replaced my readme file tinkering around… Also any suggestions on my site that would help me land a dev job would be great! I applied yesterday and they should be checking it out next week or so.</p>

**Comment by big616** [comment]
<p>I should apologize for the noob question first i guess, but please, anyone care to help?</p>

**Comment by birdcar** [comment, marked as answer]
<p>Hi <a class="mention" href="/big616">@big616</a>!</p>
<p>No need to apologize for the “noob question”, we’ve all been there at some point.</p>
<p>The reason you’re not seeing your theme show up is because of the way that Jekyll handles HTML files. Since you’re trying to get your theme to apply to your <code>index.html</code> file, you’ll need to tell Jekyll explicitly what theme to use for your HTML file.</p>
<p>To do this, delete everything from your HTML file except what’s between the <code>\&lt;body</code>&gt; tags in your repository. Your theme will take care of the other HTML, so you just need to worry about the content of the page, which will always be in the <code>\&lt;body\&gt;</code> of your theme.</p>
<p>Once you’ve done that, at the very top of your <code>index.html</code> file, you’ll need to add some Jekyll front matter to select the theme layout your page uses. Since you’re using the GitHub Pages “Dinky” theme, you’ll want to use the “default” layout.</p>
<pre><code>---
layout: default
---
</code></pre>
<p>Once you’ve added the three lines above to the top of your HTML file, you can commit your changes and your Pages site should start building again. Once it’s done being built, you should be all set. You may need to clear your browser’s cache to see the changes, but your site should now have a theme applied to it.</p>
<p>Let me know if you have any additional questions!</p>

**Reply by NetPixelX** [reply]
How do i do it any video?

**Comment by Nandy-Saran** [comment]
<p>Thanks a lot. That was really helpful.</p>

**Comment by ajinkyakolhe** [comment]
<p>Very clear instructions, worked like magic. Thank you!</p>

**Comment by reganconnell** [comment]
<p>I was having this same issue and realized that github pages only supports certain themes (<a href="https://pages.github.com/themes/" rel="nofollow noopener">https://pages.github.com/themes/</a>), however, you can specify a “remote_name:” in your _config.yml file to use a public theme not listed in the link provided. This post (<a href="https://github.blog/2017-11-29-use-any-theme-with-github-pages/" rel="nofollow noopener">https://github.blog/2017-11-29-use-any-theme-with-github-pages/</a>) explains further!</p>

**Comment by andrei-moldovan-dev** [comment]
<p>Thanks, buddy! Couldn’t find how to apply themes to my HTML pages anywhere. You’re a life saver!</p>

**Comment by karikarthick** [comment]
<p>It works. Great!</p>

**Comment by rahil627** [comment]
<p>i just wanted to add that this worked for index.md too (not just index.html).</p>
<p>I was having the same problem. I tried the <code>page</code> and <code>post</code> layout-value. Nothing. Then I changed it to <code>default</code> and voila! Amazing. :slight_smile:</p>

**Comment by landa125** [comment]
<p>I am having a similar issue. But I am using a theme that is located in theme.js but I tried adding in the HTML by href=“js/theme.js”. I still can’t get my theme to show on my portfolio website.</p>
<p>Please help me I’m a noob :frowning:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="http://oscarlanda.me/" target="_blank" rel="noopener nofollow ugc">oscarlanda.me</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="http://oscarlanda.me/" target="_blank" rel="noopener nofollow ugc">Home - Brand</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by otter-computer** [comment]
<p>It looks like your links are looking for your CSS inside the <code>assets</code> folder:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/landa125/landa125.github.io/blob/4b942dd822a3aee67031ce040276bdd54e1b6545/index.html#L17" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/landa125/landa125.github.io/blob/4b942dd822a3aee67031ce040276bdd54e1b6545/index.html#L17" target="_blank" rel="noopener">landa125/landa125.github.io/blob/4b942dd822a3aee67031ce040276bdd54e1b6545/index.html#L17</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="7" style="counter-reset: li-counter 6 ;">
<li>    &lt;title&gt;Home - Brand&lt;/title&gt;</li>
<li>    </li>
<li>     &lt;!-- Bootstrap core CSS --&gt;</li>
<li>    &lt;link rel="stylesheet" href="fonts/ionicons.min.css"&gt;</li>
<li>    &lt;link rel="stylesheet" href="fonts/material-icons.min.css"&gt;</li>
<li>    &lt;link href="css/bootstrap.css" rel="stylesheet"&gt;</li>
<li>
</li>
<li>    &lt;!-- Custom CSS for the '3 Col Portfolio' Template --&gt;</li>
<li>    &lt;link href="js/theme.js"&gt;</li>
<li>    </li>
<li class="selected">    &lt;link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css"&gt;</li>
<li>    &lt;link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,400,700"&gt;</li>
<li>    &lt;link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Aldrich"&gt;</li>
<li>    &lt;link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Alfa+Slab+One"&gt;</li>
<li>    </li>
<li>    &lt;link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pikaday/1.6.1/css/pikaday.min.css"&gt;</li>
<li>&lt;/head&gt;</li>
<li>
</li>
<li>&lt;body&gt;</li>
<li>    &lt;nav class="navbar navbar-dark navbar-expand-lg fixed-top bg-white portfolio-navbar gradient" style="font-family: Aldrich, sans-serif;"&gt;</li>
<li>        &lt;div class="container"&gt;&lt;a class="navbar-brand logo" href="#"&gt;Portfolio&lt;/a&gt;&lt;button data-toggle="collapse" class="navbar-toggler" data-target="#navbarNav"&gt;&lt;span class="sr-only"&gt;Toggle navigation&lt;/span&gt;&lt;span class="navbar-toggler-icon"&gt;&lt;/span&gt;&lt;/button&gt;</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>But that folder doesn’t exist inside your repository. This link should probably be this instead:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="css/bootstrap.min.css"&gt;
</code></pre>
<p>You’ll probably need to fix these path issues with any other assets on your site that aren’t loading too.</p>
